cev_eris.dme

```diff
@@ -1,2790 +1,2790 @@
-#include "code\game\gamemodes\events\hidden_events\bluespace_rift.dm"
-#include "code\game\gamemodes\events\hidden_events\kaiser.dm"
-#include "code\game\gamemodes\events\holidays\Christmas.dm"
-#include "code\game\gamemodes\events\holidays\Holidays.dm"
-#include "code\game\gamemodes\events\weather\rad_storm.dm"
-#include "code\game\gamemodes\malfunction\malf_hardware.dm"
-#include "code\game\gamemodes\malfunction\malf_research.dm"
-#include "code\game\gamemodes\malfunction\malf_research_ability.dm"
-#include "code\game\gamemodes\malfunction\newmalf_ability_trees\HARDWARE.dm"
-#include "code\game\gamemodes\malfunction\newmalf_ability_trees\HELPERS.dm"
-#include "code\game\gamemodes\malfunction\newmalf_ability_trees\tree_interdiction.dm"
-#include "code\game\gamemodes\malfunction\newmalf_ability_trees\tree_manipulation.dm"
-#include "code\game\gamemodes\malfunction\newmalf_ability_trees\tree_networking.dm"
-#include "code\game\gamemodes\nuclear\pinpointer.dm"
-#include "code\game\gamemodes\roleset\roleset.dm"
-#include "code\game\gamemodes\roleset\simple.dm"
-#include "code\game\gamemodes\roleset\faction\excelsior.dm"
-#include "code\game\gamemodes\roleset\faction\mercenary.dm"
-#include "code\game\gamemodes\roleset\faction\roleset_faction.dm"
-#include "code\game\gamemodes\storytellers\chronicler.dm"
-#include "code\game\gamemodes\storytellers\guide.dm"
-#include "code\game\gamemodes\storytellers\healer.dm"
-#include "code\game\gamemodes\storytellers\jester.dm"
-#include "code\game\gamemodes\storytellers\mime.dm"
-#include "code\game\gamemodes\storytellers\sleeper.dm"
-#include "code\game\gamemodes\storytellers\warrior.dm"
-#include "code\game\jobs\_access_defs.dm"
-#include "code\game\jobs\access.dm"
-#include "code\game\jobs\access_datum.dm"
-#include "code\game\jobs\department.dm"
-#include "code\game\jobs\jobs.dm"
-#include "code\game\jobs\wages.dm"
-#include "code\game\jobs\whitelist.dm"
-#include "code\game\jobs\job\assistant.dm"
-#include "code\game\jobs\job\assistant_flavors.dm"
-#include "code\game\jobs\job\captain.dm"
-#include "code\game\jobs\job\church.dm"
-#include "code\game\jobs\job\civilian.dm"
-#include "code\game\jobs\job\engineering.dm"
-#include "code\game\jobs\job\guild.dm"
-#include "code\game\jobs\job\job.dm"
-#include "code\game\jobs\job\medical.dm"
-#include "code\game\jobs\job\science.dm"
-#include "code\game\jobs\job\security.dm"
-#include "code\game\jobs\job\silicon.dm"
-#include "code\game\machinery\adv_med.dm"
-#include "code\game\machinery\ai_slipper.dm"
-#include "code\game\machinery\alarm.dm"
-#include "code\game\machinery\antigrav.dm"
-#include "code\game\machinery\atmo_control.dm"
-#include "code\game\machinery\autolathe_disk_cloner.dm"
-#include "code\game\machinery\Beacon.dm"
-#include "code\game\machinery\biogenerator.dm"
-#include "code\game\machinery\bioprinter.dm"
-#include "code\game\machinery\bioprinter_nt.dm"
-#include "code\game\machinery\bluespaceartillery.dm"

```

Comment:
Cringe

Reply:
crlf -> lf

Reply:
Why we need to get from crlf to lf if this PR just replaces Travis with actions?